pyproject.toml

```diff
@@ -28,14 +28,16 @@ pytest-xdist = "*"
 filelock = "^3.0"
 pytest-ordering = "*"
 lovely-pytest-docker = { version="^0", optional = true }
-junitparser = "<3.0.0"
+junitparser = "<2.0.0"
```

Comment:
@rajkarkar-crest Why are we downgrading this library?

Reply:
Version 2+ of junitparser is breaking an existing feature, upgrading the library will be in the future.